docs/src/modules/java/pages/call-another-service.adoc

```diff
@@ -0,0 +1,59 @@
+= Calling other services
+
+// FIXME this does not actually work (yet?) https://github.com/lightbend/akkaserverless-docs/issues/903#issuecomment-919174111
+
+== Akka Serverless services
+// FIXME possible from the same project only, or cross projects as well?
```

Comment:
I think it's just in the same project / namespace.

Reply:
Cross project communication is blocked by firewall rules.

Reply:
Thanks for verifying, I added that for cross project you have to consume the public API/routes of the service.